department-of-veterans-affairs/caseflow
# janet/APPEALS-8610 enqueue send notification job in appellant notification module

Resolves [APPEALS-8610](https://vajira.max.gov/browse/APPEALS-8610)

### Description
The notification module has logic that will enqueue the send notification method.

### Acceptance Criteria
- [x] AppellantNotification module calls SendNotificationJob to queue payload
- [x] Rspec tests are updated and passing

### Testing Plan
1. Open two terminals
2. Run `bundle exec shoryuken -q caseflow_development_send_notifications -R` in one terminal
3. Run `appeal=Appeal.first` in other terminal
4. Run `AppellantNotification.notify_appellant(appeal, "Hearing scheduled")` 
5. Monitor the output in the terminal where shoryuken is running. It should look like this:
![sjfieojdlk](https://user-images.githubusercontent.com/93942793/187572649-8770182c-10ce-4172-a70b-c23663982b4c.PNG)
